From eccbc528d9800bdfef5e1795e6bada446aafaf43 Mon Sep 17 00:00:00 2001
From: Lumir Balhar <lbalhar@redhat.com>
Date: Thu, 16 May 2019 13:50:14 +0200
Subject: [PATCH] Do not turn DeprecationWarning into Exception

---
 tornado/test/runtests.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tornado/test/runtests.py b/tornado/test/runtests.py
index e4f938d3..cd2a5b87 100644
--- a/tornado/test/runtests.py
+++ b/tornado/test/runtests.py
@@ -127,8 +127,6 @@ def main():
     # Tornado generally shouldn't use anything deprecated, but some of
     # our dependencies do (last match wins).
     warnings.filterwarnings("ignore", category=DeprecationWarning)
-    warnings.filterwarnings("error", category=DeprecationWarning,
-                            module=r"tornado\..*")
     warnings.filterwarnings("ignore", category=PendingDeprecationWarning)
     warnings.filterwarnings("error", category=PendingDeprecationWarning,
                             module=r"tornado\..*")
-- 
2.21.0

